pkg/bpf/prog.go

```diff
@@ -67,3 +74,72 @@ func (t ProgType) String() string {
 
 	return "Unknown"
 }
+
+type attrProg struct {
+	startID   uint32
+	progID    uint32
+	nextID    uint32
+	openFlags uint32
+}
+
+// ProgInfo holds values from the upstram struct bpf_prog_info.
+// From: https://github.com/torvalds/linux/blob/master/include/uapi/linux/bpf.h#L1006
+type ProgInfo struct {
+	ProgType     ProgType
+	ID           uint32
+	MapIDs       uint64
+	CreatedByUID uint32
+	LoadTime     uint64 /* ns since boottime */
+	Name         string
+}
+
+type attrObjInfo struct {
```

Comment:
Somehow I dropped my comment here. Similar to the above about `union bpf_attr`, I think this is the link:

https://github.com/torvalds/linux/blob/v4.16/include/uapi/linux/bpf.h#L309